docs/api/api_blueprint.apib

```diff
@@ -12009,6 +12009,62 @@ Get a list of Notifications.
         }
 
 
+## Notifications Admin Collection [/v1/notifications{?page%5bsize%5d,page%5bnumber%5d,sort,filter}]
++ Parameters
+    + page%5bsize%5d (optional, integer, `10`) - Maximum number of resources in a single paginated response.
+    + page%5bnumber%5d (optional, integer, `2`) - Page number to fetched for the paginated response.
+    + sort (optional, string, `message`) - Sort the resources according to the given attribute in ascending order. Append '-' to sort in descending order.
+    + filter (optional, string, `[]`) - Filter according to the flask-rest-jsonapi filtering system. Please refer: http://flask-rest-jsonapi.readthedocs.io/en/latest/filtering.html for more.
+
+### List All Notifications [GET]
+Get a list of Notifications.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+
+            Authorization: JWT <Auth Key>
+
++ Response 200 (application/vnd.api+json)
+
+        {
+            "meta": {
+                "count": 1
+            },
+            "data": [
+                {
+                    "relationships": {
+                        "user": {
+                            "links": {
+                                "self": "/v1/notifications/1/relationships/user",
+                                "related": "/v1/notifications/1/user"
+                            }
+                        }
+                    },
+                    "attributes": {
+                        "message": "example",
+                        "received-at": "2017-06-23T04:40:37.239724+00:00",
+                        "is-read": false,
+                        "title": "example"
+                    },
+                    "type": "notification",
+                    "id": "1",
+                    "links": {
+                        "self": "/v1/notifications/1"
+                    }
+                }
+            ],
+            "jsonapi": {
+                "version": "1.0"
+            },
+            "links": {
+                "self": "/v1/users/2/notifications"
```

Comment:
Can you please check the self url here once again? I haven't checked it yet but I think it should be `/v1/notifications`.